runtime/doc/lsp.txt

```diff
@@ -806,6 +806,13 @@ get_log_path()                                        *vim.lsp.get_log_path()*
     Return: ~
         (string) path to log file
 
+inlay_hint({bufnr}, {enable})                           *vim.lsp.inlay_hint()*
+    Enable/disable/toggle inlay hints for a buffer
+
+    Parameters: ~
+      • {bufnr}   (integer) Buffer handle, or 0 for current
```

Comment:
I also think we should follow up and allow `nil` here, to allow users to toggle it for all open & future buffers at once - which reinforces the move out of vim.lsp.buf.